src/index.js

```diff
@@ -67,7 +67,7 @@ function applyRef (e, ref) {
 function ensureAttrs (objs) {
   const { attrs, events, ref, key, ...props } = objs || {};
   const newRef = ensureRef({ attrs, events, props, ref });
-  return { ref: newRef, ...(key ? { key } : {}) }
```

Comment:
Wouldn't doing mean that a `key` attribute is always present on all markup received by React, regardless if it is defined or not? Perhaps that isn't an issue, but I didn't want to take the chance. If you think that is safe, I'll update it.

I just tried it out with `return { ref: newRef, key }` and it nothing seemed to break. So maybe React is tolerant of `key` attributes on everything.